src/test/ui/closures/closure_no_cap_coerce_many_unsafe_1.rs

```diff
@@ -0,0 +1,23 @@
+// run-pass
```

Comment:
Can there be run-pass tests that along the lines of
```
let funcs = [add, |a, b| (a - b) as i32];
assert_eq!([funcs[0](5, 5), funcs[1](5, 5)], [10, 0]);
```

Reply:
Added, you can check `src/test/ui/closures/closure_no_cap_coerce_many_run_pass.rs`